emeis/settings.py

```diff
@@ -243,6 +245,10 @@ def parse_admins(admins):
 
 ADMINS = parse_admins(env.list("ADMINS", default=[]))
 
+USER_EXPORT_PATH = env.str(
```

Comment:
I suggest renaming it to `USER_EXPORT_TEMPLATE_PATH`. Also, please add a line about this to the README.